### PR TITLE
Nolt sso (along with a better nav dropdown)

### PR DESCRIPTION
**Pull request by ukutaht**
### Changes

1. Adds SSO login capability for Nolt to take feature requests seamlessly
2. Uses a nicer nav dropdown based on a TailwindUI component
3. Moves `Docs` link to dropdown as `Help Center`, and adds a new `Contact Support` link
4. Upgrades Alpine.js for the dropdown

Demo: https://www.loom.com/share/1b6d273052b146668d043c01fbea089e

### Tests
- [x] This PR does not require tests

### Changelog
- [x] Entry has been added to changelog

### Documentation
- [x] This change does not need a documentation update

### Dark mode
- [x] The UI has been tested both in dark and light mode

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -2,11 +2,15 @@ import "../css/app.css"
...
 import 'abortcontroller-polyfill/dist/polyfill-patch-fetch'
-import 'alpinejs'
+import Alpine from 'alpinejs'
 import "./liveview/live_socket"
 import "./liveview/suggestions_dropdown"
+import "./liveview/dropdown"
 import "./liveview/phx_events"
 
+
+Alpine.start()
```
Comment (ukutaht) [inline assets/js/app.js:12]: https://alpinejs.dev/upgrade-guide#need-to-call-alpine-start

File: assets/js/liveview/dropdown.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// From https://alpinejs.dev/component/dropdown
+//
+
+window.dropdown = {
```
Comment (ukutaht) [inline assets/js/liveview/dropdown.js:4]: TODO: Use https://alpinejs.dev/globals/alpine-data instead

**Comment by aerosol** [comment]
cool! the dropdown looks really nice, although perhaps gravatar integration should be opt-in 😬

**Review by aerosol — APPROVED** [review]

File: lib/plausible_web/controllers/avatar_controller.ex [hunk 3]
```diff
@@ -0,0 +1,38 @@
+defmodule PlausibleWeb.AvatarController do
+  @moduledoc """
+  This module proxies requests to BASE_URL/avatar/:hash to www.gravatar.com/avatar/:hash.
+
+  The purpose is to make use of Gravatar's convenient avatar service without exposing information
+  that could be used for tracking the Plausible user. Comapred to requesting the Gravatar directly
```
Comment (aerosol) [inline lib/plausible_web/controllers/avatar_controller.ex:6]:
```suggestion
  that could be used for tracking the Plausible user. Compared to requesting the Gravatar directly
```

File: assets/css/app.css [hunk 4]
```diff
@@ -41,6 +41,7 @@ html {
 body {
 	-webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  width:100vw;  /* Prevents content from jumping when scrollbar is added/removed due to vertical oveflow */
```
Comment (aerosol) [inline assets/css/app.css:44]:
```suggestion
  width:100vw;  /* Prevents content from jumping when scrollbar is added/removed due to vertical overflow */
```

**Pull request merged**